conformance/lang/expressions/xml-attribute-access-expr/xml_attribute_access_expr.balt

```diff
@@ -0,0 +1,447 @@
+Test-Case: output
+Description: Test xml-required-attribute-access-expr when xml-attribute-name is an identifier and
+             expression has only a root element.
+Labels: xml, xml-required-attribute-access-expr, string, xml-filter-expr
+
+function init() {
+    xml x = xml `<root attr="attr-val"></root>`;
+    io:println(x.attr); // @output attr-val
+    io:println(x.<root>.attr); // @output attr-val
```

Comment:
This can still be split into separate tests? Basically a separate test on just a variable reference, a separate one on xml-filter-expr, etc.

Reply:
Changed in all places